From 712bde033ef8eab718041efb7e94d1e2ba6e4cb6 Mon Sep 17 00:00:00 2001
From: ArenM <aren@peacevolution.org>
Date: Thu, 4 May 2023 21:19:41 -0400
Subject: [PATCH sxmo-utils 2/6] Fixup icons for nerd fonts 3.0

Nerd fonts 3.0 removes a bunch of these code points because they weren't
actually in the private use ares. See the release notes for nerd fonts
https://www.nerdfonts.com/releases.

I also checked nerd fonts 2.3.3 to make sure this won't break icons
there, it doesn't.
---
 configs/default_hooks/sxmo_hook_icons.sh | 90 ++++++++++++------------
 1 file changed, 45 insertions(+), 45 deletions(-)

diff --git a/configs/default_hooks/sxmo_hook_icons.sh b/configs/default_hooks/sxmo_hook_icons.sh
index d19d4d5..c86fac8 100755
--- a/configs/default_hooks/sxmo_hook_icons.sh
+++ b/configs/default_hooks/sxmo_hook_icons.sh
@@ -14,15 +14,15 @@
 
 icon_a2x="⇄"
 icon_a2y="⇅"
-icon_ac1=""
-icon_ac2=""
-icon_ac3=""
-icon_ac4=""
+icon_ac1="󰁛"
+icon_ac2="󰁜"
+icon_ac3="󰁃"
+icon_ac4="󰁂"
 icon_and=""
-icon_ard=""
-icon_arl=""
-icon_arr=""
-icon_aru=""
+icon_ard="󰁅"
+icon_arl="󰁍"
+icon_arr="󰁔"
+icon_aru="󰁝"
 icon_att=""
 icon_bel=""
 icon_bok=""
@@ -32,38 +32,38 @@ icon_cam=""
 icon_cfg="" #configuration cog
 icon_chk=""
 icon_chs="♜" #chess
-icon_clc="" #calculator
-icon_clk=""
-icon_cls="" #close
+icon_clc="󰃬" #calculator
+icon_clk="󰥔"
+icon_cls="󰅖" #close
 icon_com="" # (Laptop) Computer (💻)
 icon_cpy=""
-icon_del="﫧" #delete
+icon_del="󰅖" #delete
 icon_dir="" #directory folder
 icon_dof="" # dot off
 icon_don="" # dot on
 icon_dop="" # dot point
-icon_dot="" # searching, connecting, etc.
-icon_drw="" # Drawing tablet
+icon_dot="󰇘" # searching, connecting, etc.
+icon_drw="󰏬" # Drawing tablet
 icon_ear="" # earpiece
-icon_edt="" #editor
-icon_eml="" #email
-icon_exp="" #expand
+icon_edt="󰎞" #editor
+icon_eml="󰇮" #email
+icon_exp="󰁌" #expand
 icon_fbw=""
 icon_ffw="" #fast forward
 icon_ffx="" #firefox
 icon_fil="" #file
 icon_flk="" #falkon
-icon_fll="" #flashlight
+icon_fll="󰉄" #flashlight
 icon_flt="" #filter/pipe
 icon_fnd="" #search/find
-icon_gam="" # gaming controller (🎮)
+icon_gam="󰊴" # gaming controller (🎮)
 icon_glb="" #globe
 icon_gps='' #gps
-icon_grd=""
+icon_grd="󰀻"
 icon_grp="" #group
 icon_hdp="" #headphones
 icon_hom=""
-icon_img=""
+icon_img="󰏜"
 icon_inf="" #info
 icon_itm="" #item (default)
 icon_kbd="" #keyboard
@@ -71,26 +71,26 @@ icon_key=""
 icon_lck=""
 icon_lnk=""
 icon_lst="" #list
-icon_map=""
-icon_mdd="" # modem disabled state
-icon_mic=""
-icon_mmc=""
+icon_map="󰍍"
+icon_mdd="󰄢" # modem disabled state
+icon_mic="󰍬"
+icon_mmc="󰍭"
 icon_mnu=""
 icon_mod="" # modem
-icon_mov=""
-icon_mse="" # computer mouse (🖱️)
-icon_msg="" #text
-icon_mus=""
+icon_mov="󰁁"
+icon_mse="󰍽" # computer mouse (🖱️)
+icon_msg="󰍦" #text
+icon_mus="󰎈"
 icon_mut="" #mute
-icon_mvi="" #movie
-icon_net=""
+icon_mvi="󰎁" #movie
+icon_net="󰀂"
 icon_nto="" #no touch
 icon_nxt="" #next
-icon_out="" #logout
+icon_out="󰍃" #logout
 icon_pau=""
-icon_phl="" #phonelog
-icon_phn="" #phone
-icon_phx="" #hangup
+icon_phl="󰏹" #phonelog
+icon_phn="󰏲" #phone
+icon_phx="󰏵" #hangup
 icon_plk="" # phone locked
 icon_pls=""
 icon_prn="🖶" # printer (🖨️)
@@ -98,14 +98,14 @@ icon_prv="" #previous
 icon_pst=""
 icon_pwr="⏻" #power
 icon_red="" #reddit
-icon_ret="" #return
+icon_ret="󰌑" #return
 icon_rld=""
 icon_rol="" #rotate left
 icon_ror="" #rotate right
 icon_rss=""
-icon_sav="" #save
+icon_sav="󰆓" #save
 icon_sfl="" #shuffle, random
-icon_shr="" #shrink
+icon_shr="󰁄" #shrink
 icon_snd="" #send
 icon_spk="" #speaker
 icon_spl=""
@@ -114,19 +114,19 @@ icon_stp="" #stop
 icon_str="" #star
 icon_tab=""
 icon_tgm='' #telegram
-icon_tmr="" #timer
+icon_tmr="󰀠" #timer
 icon_tof=""
 icon_ton=""
 icon_trh="" # trash
 icon_trm='' #terminal
-icon_upc="" #up in circle
+icon_upc="󰁠" #up in circle
 icon_usb="" # usb
-icon_usr=""
+icon_usr="󰀄"
 icon_vid="" #video
 icon_vim=''
 icon_wif=""
-icon_wfo="睊" # wifi off
-icon_wfh="" # wifi hotspot
+icon_wfo="󰖪" # wifi off
+icon_wfh="󰀂" # wifi hotspot
 icon_win="" #window
 icon_wat="" # watch (⌚)
 icon_wn2=""
@@ -160,7 +160,7 @@ icon_modem_threeg="󰜓" # a lot (see sxmo_hook_statusbar.sh)
 icon_modem_hspa="󰜕" # hspa
 icon_modem_hspa_plus="󰜖" # hspa plus
 icon_modem_twog="󰜒" # edge, pots, gsm, gprs, etc.
-icon_modem_notech="ﲁ" # disabled cell bars
+icon_modem_notech="󰞃" # disabled cell bars
 
 # modem signal strengths
 icon_modem_signal_0="󰢿"
@@ -186,7 +186,7 @@ icon_wifi_disconnected="󰤮"
 icon_bat_c_0="󰢟"
 icon_bat_c_1="󱊤"
 icon_bat_c_2="󱊥"
-icon_bat_c_3=""
+icon_bat_c_3="󰂅"
 icon_bat_0="󰂎"
 icon_bat_1="󱊡"
 icon_bat_2="󱊢"
-- 
2.40.0

